RecoParticleFlow/PFClusterProducer/plugins/alpaka/PFMultiDepthClusterizer_Alpaka.dev.cc

```diff
@@ -0,0 +1,186 @@
+#include <Eigen/Core>
+#include <Eigen/Dense>
```

Comment:
Eigen does not seem to be used in this file, are these includes needed here or can they be removed?